src/Controller/PageAdminController.php

```diff
@@ -54,6 +54,8 @@ public static function getSubscribedServices(): array
     }
 
     /**
+     * @param ProxyQueryInterface<object> $query
```

Comment:
But if we know this is batch for pages, shouldnt this objet be PageInterface?

cc @VincentLanglet

Reply:
Indeed